<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wallet</class>
 <widget class="QDialog" name="Wallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartShopXpress</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/img0/logo3.png</normaloff>:/img/img0/logo3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/img/img0/gbackground.png);</string>
  </property>
  <widget class="QLabel" name="walletLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/walletimg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/menuicon.png);
border-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="moneyimglabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
border-image: url(:/img/img0/moneyimg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentMoneyLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border-image: url(:/img/img0/lineEditBackground.png);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#21561e;&quot;&gt;0,00$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="moneyimglabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
border-image: url(:/img/img0/card_no.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="moneyimglabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
border-image: url(:/img/img0/valid_thru.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="moneyimglabel_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
border-image: url(:/img/img0/sec_code.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border-image: url(:/img/img0/amountofmoneyimg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/img/img0/onay.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="monthComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <item>
    <property name="text">
     <string>--</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>03</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>04</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>06</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>07</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="cardNoLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>231</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/carddetailimg.png);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="yearComboBox">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>390</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <item>
    <property name="text">
     <string>--</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2025</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2026</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2027</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2028</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2029</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2030</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2031</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2032</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2033</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2034</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2035</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2036</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2037</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2038</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2038</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2039</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2040</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2041</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2042</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2043</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2044</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2045</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2046</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2047</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2048</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2049</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2050</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="moneySpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="securityCodeSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
